### PR TITLE
feat: 1inch swapper pre-release first pass fixes

### PR DESCRIPTION
**Pull request by woodenfurniture**
## Description

Adds initial fixes and chores to 1inch swapper that is yet to be tested by ops. This is a first pass set of changes for review while I'm asleep :)

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

NA

## Risk

no user-facing risk

## Testing

So far these changes are only focused on trade input state and quotes. Trades are yet to be tested.

### Engineering

See above

### Operations

No action required.

### CODE REVIEW

**Comment by woodenfurniture** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #4395** <a href="https://app.graphite.dev/github/pr/shapeshift/web/4395" target="_blank">Graphite</a> 
    * **PR #4396** <a href="https://app.graphite.dev/github/pr/shapeshift/web/4396" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/4396?utm_source=stack-comment).

File: .env.base [hunk 1]
```diff
@@ -5,38 +5,39 @@ REACT_APP_FEATURE_ARKEO_AIRDROP=true
...
+REACT_APP_FEATURE_COWSWAP=true
+REACT_APP_FEATURE_LIFI_SWAP=false
 REACT_APP_FEATURE_ONE_INCH=false
+REACT_APP_FEATURE_OSMOSIS_SWAP=true
+REACT_APP_FEATURE_THOR_SWAP=true
+REACT_APP_FEATURE_ZRX_AVALANCHE=true
+REACT_APP_FEATURE_ZRX_BNBSMARTCHAIN=true
+REACT_APP_FEATURE_ZRX_ETHEREUM=true
+REACT_APP_FEATURE_ZRX_OPTIMISM=true
+REACT_APP_FEATURE_ZRX_POLYGON=true
```
Comment (gomesalexandre) [inline .env.base:40]: ty, confirmed those are just moved around

File: src/lib/swapper/swappers/utils/constants.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
 export const APPROVAL_GAS_LIMIT = '100000' // Most approvals are around 40k, we've seen 72k in the wild, so 100000 for safety.
 export const DEFAULT_SLIPPAGE = '0.002' // .2%
 export const ALLOWABLE_MARKET_MOVEMENT = '0.01' // 1%
+export const MAX_ALLOWANCE = '100000000000000000000000000'
```
Comment (gomesalexandre) [inline src/lib/swapper/swappers/utils/constants.ts:4]: Can also remove from THOR, ZRX and CoW utils and consume the common one while at it

**Review by gomesalexandre — DISMISSED** [review]
Untested but conceptually LGTM - added one comment WRT further cleanup, but happy to get this in so we can get it ops re-tested

**Review by gomesalexandre — APPROVED** [review]
Restamp after cleanup and stack edit

File: src/lib/swapper/swappers/utils/constants.ts (hunk 2 above)
Comment (gomesalexandre) [reply src/lib/swapper/swappers/utils/constants.ts:4]: https://github.com/shapeshift/web/pull/4396/commits/d58b8db4f8e7271b294bbbf3e02c13465b0369f9

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**